### PR TITLE
Playground runners scala update

### LINKED ISSUE #24872: [Tour of Beam] [Task] Support scio examples
### What needs to happen?
Need to add support for scio examples in playground.
This requires whitelisting maven repo host

### Issue Priority
Priority: 3

### Issue Component
Component: beam-playground, tour-of-beam

### PR DESCRIPTION
**Pull request by olehborysevych**
Whitelisting maven repo for sbt to work
Externalizing grpc timeout
Adding example deployment to readme
fixes #24872

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by olehborysevych** [comment]
R: @damccorm

File: playground/README.md [hunk 1]
```diff
@@ -120,3 +120,50 @@ cd beam
...
 See [terraform](./terraform/README.md) for details on how to build and deploy
 the application and its dependent infrastructure.
+
+# Manual Example deployment
+
+The following requirements are needed for deploying examples manually.
+
+1. GCP project with deployed Playground backend
+2. Python (3.9.x)
+3. Login into GCP (gcloud deafult login or using service account key)
```
Comment (damccorm) [inline playground/README.md:130]:
```suggestion
3. Login into GCP (gcloud default login or using service account key)
```

Nit

**Review by damccorm — APPROVED** [review]

**Comment by damccorm** [comment]
LGTM, thanks! Just one nit

File: playground/README.md [hunk 2]
```diff
@@ -120,3 +120,50 @@ cd beam
...
 See [terraform](./terraform/README.md) for details on how to build and deploy
 the application and its dependent infrastructure.
+
+# Manual Example deployment
+
+The following requirements are needed for deploying examples manually:
+
+1. GCP project with deployed Playground backend
+2. Python (3.9.x)
+3. Login into GCP (gcloud deafault login or using service account key)
```
Comment (damccorm) [inline playground/README.md:130]:
```suggestion
3. Login into GCP (gcloud default login or using service account key)
```

Nit - looks like an extra `a` still slipped in

**Pull request merged**